kgtk/visualize/visualize_api.py

```diff
@@ -664,11 +640,12 @@ def __init__(
             show_text_limit: int = 500,
             node_border_color: str = None,
             tooltip_column: str = None,
-            text_node: str = None,
+            text_node: str = 'false',
```

Comment:
again, instead of using string false, why not use boolean `False`

Reply:
changed to boolean False